jsk_arc2017_baxter/euslisp/lib/arc-interface.l

```diff
@@ -335,7 +336,9 @@
                   (- (/ (m->mm (send bin-box :dimensions :x)) 2.0))
                   0.0 (/ (m->mm (send bin-box :dimensions :z)) 2.0))
                 offset) :world)
-      (send bin-coords :rotate (* pi (/ coords-pitch 180)) :y)
+      (send bin-coords :rotate (deg2rad (aref rpy 1)) :y)
+      (send bin-coords :rotate (deg2rad (aref rpy 2)) :z)
+      (send bin-coords :rotate (deg2rad (aref rpy 0)) :x)
```

Comment:
And rotating order is `roll-pitch-yaw`:
```
14.irteusgl$ setq c (make-coords :rpy #f(1 1 1))
#<coordinates #X5105708  0.0 0.0 0.0 / 1.0 1.0 1.0>
15.irteusgl$ objects (list a b c)
(#<body #X493f8e0 (:cube 100.0 100.0 100.0) 0.0 0.0 0.0 / 0.0 0.0 0.0> #<coordinates #X4adeb98  0.0 0.0 0.0 / 0.0 0.0 0.0> #<coordinates #X5105708  0.0 0.0 0.0 / 1.0 1.0 1.0>)
16.irteusgl$ send c :scale 5
#<coordinates #X5105708  0.0 0.0 0.0 / 1.0 1.0 1.0>
```
![screenshot from 2017-05-20 11 27 34](https://cloud.githubusercontent.com/assets/14994939/26272383/c2f193a4-3d51-11e7-8f76-c760a3404139.png)
```
31.irteusgl$ send b :rotate 1 :x :world
#<coordinates #X5242d58  0.0 0.0 0.0 / 0.0 0.0 1.0>
32.irteusgl$ send b :rotate 1 :y :world
#<coordinates #X5242d58  0.0 0.0 0.0 / 0.0 1.0 1.0>
33.irteusgl$ send b :rotate 1 :z :world
#<coordinates #X5242d58  0.0 0.0 0.0 / 1.0 1.0 1.0>
```
![screenshot from 2017-05-20 11 32 12](https://cloud.githubusercontent.com/assets/14994939/26272390/e5618980-3d51-11e7-8c36-10d8f3ed64bc.png)
```
34.irteusgl$ setq b (make-coords)
#<coordinates #X52128b8  0.0 0.0 0.0 / 0.0 0.0 0.0>
35.irteusgl$ send b :scale 6
#<coordinates #X52128b8  0.0 0.0 0.0 / 0.0 0.0 0.0>
36.irteusgl$ objects (list a b c)
(#<body #X493f8e0 (:cube 100.0 100.0 100.0) 0.0 0.0 0.0 / 0.0 0.0 0.0> #<coordinates #X52128b8  0.0 0.0 0.0 / 0.0 0.0 0.0> #<coordinates #X5105708  0.0 0.0 0.0 / 1.0 1.0 1.0>)
37.irteusgl$ send b :rotate 1 :z :world
#<coordinates #X52128b8  0.0 0.0 0.0 / 1.0 0.0 0.0>
38.irteusgl$ send b :rotate 1 :y :world
#<coordinates #X52128b8  0.0 0.0 0.0 / 1.237 0.472 0.919>
39.irteusgl$ send b :rotate 1 :x :world
#<coordinates #X52128b8  0.0 0.0 0.0 / 1.235 -0.481 1.235>
```
![screenshot from 2017-05-20 11 33 25](https://cloud.githubusercontent.com/assets/14994939/26272405/2b321f38-3d52-11e7-8d60-a9d3a24a897f.png)